requirements.txt

```diff
@@ -4,6 +4,7 @@ scipy
 phe
 line_profiler
 clint
+coverage
```

Comment:
Hey, I think `line_profiler`, `clint`, `coverage`, `pytest` and `pytest-flake8` should be moved to `test-requirements`. While `sklearn` should go all the way to `dev-requirements`

Reply:
Where are the dev-requirements?

Reply:
In `dev-requirements.txt`, but maybe you have to pull from `master` because it's been added quite recently by #270

Reply:
@baldassarreFe done.